src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java

```diff
@@ -71,13 +87,15 @@ public void mapEnvelope_maps_all_fields_correctly() {
         assertThat(exceptionRecord.envelopeLegacyCaseReference).isEqualTo(envelope.legacyCaseRef);
         assertThat(exceptionRecord.showEnvelopeCaseReference).isEqualTo("No"); // for "New Application"
         assertThat(exceptionRecord.showEnvelopeLegacyCaseReference).isEqualTo("No"); // for "New Application"
+        assertThat(exceptionRecord.surname).isEqualTo("surname1");
```

Comment:
either pass `"surname1"` to `envelopeWithSurname(2);` or retrieve surname field from `envelope` object.